packages/styles/src/core/variables/_variables.scss

```diff
@@ -1,2 +1,8 @@
 @import '_colors';
 @import '_breakpoints';
+
+$u1: 4px;
```

Comment:
😞 I don't like this at all. This is hard to understand.

Reply:
What you don't like? The naming?

Reply:
Yes, the naming. I'd say `$base-size`, or `$base-grid-size`, is better.
Anything else that explain better what that variable is.